samples/monster_generated.h

```diff
@@ -25,6 +25,14 @@ enum Color {
   Color_MAX = Color_Blue
 };
 
+inline Color (&EnumValuesColor())[3] {
```

Comment:
Color* would lose the size information. The current implementation also provides access to the number of enumeration values in a type by using:
#define countof(arr) sizeof(arr) / sizeof(arr[0])
size_t colors = countof(EnumValuesColor());